### ISSUE TITLE
Popout Manager goes "cpu rouge"

### ISSUE THREAD

**Issue by dvhowell1106** [hawkeye-stan/msfs-popout-panel-manager#96]
Hello,  I started using PM a few weeks ago and find it a wonderful addon.  I am using Space Desk on a Fire HD tablet.  Most of the time it works fine, but on at least three occasions Windows Task Manager shows that the app is using 89% CPU and it hangs my computer -- can't even end task or close any other apps.  I have to resort to a hard power button shut down to recover and start my simming all over.  Needless to say this is very frustrating.  As of now the only plane I use with PM is the FlightFX Honda Jet HA-420; if that is of any importance.  I'm running Version 4.0.2.  Thanks a lot.

**Comment by hawkeye-stan** [comment]
Please try to increase "touch down touch up" delay to 5ms or high (but still get a touch response) and/or increase "refocus game window" delay to 2 seconds in POPM preference settings.

**Comment by skypilotYTS** [comment]
I just started having this same issue, use to get the freeze / crash but that went away with the last fix, now I seem to get issues when using the new PMDG 737 tablet as a pop out. I will try the fix mentioned above as that is probably what it is as the tablet requires allot of quick touches and scrolling / swiping.

**Comment by hawkeye-stan** [comment]
Please download latest 4.0.3 beta-3. I believe I have found the issue that is causing the rogue CPU runaway issue. I've also added better support for PMDG 737 EFB. Please try out the latest beta and hopefully it will resolve your issue.

Stanley

**Comment by skypilotYTS** [comment]
Thank you sir, loading it up now and will test

**Comment by skypilotYTS** [comment]
I gave it a try on 2 flights last night, certainly better, didn't see the Rouge CPU but it did freeze still. The freeze occurs when "scrolling" with finger. 

I will say it last much longer before the freeze this time.

**Comment by hawkeye-stan** [comment]
Do you mean freezing of POPM or the game?

**Comment by skypilotYTS** [comment]
So the way I have the tabley is popped out to an iPad using SpaceDesk. 

When the freeze occur (which is far less often now, so great progress!) And yes, no CPU runways anymore either) it occurs when on say the flight plan page (OFP) where you would use your finger to swipe / scroll up and down. 

When it freezes, you can still use the in-cockpit tablet, but have to use Task Manager to kill POPM as it's frozen. Then re-launch and re pop out.

**Comment by hawkeye-stan** [comment]
I've tried to replicate your issue and was only able to reproduce freezing once during couple hours of non-stop testing. And the one time was the code for PMDG EFB tried to catch up to the speed of POPM as I was swiping on the tablet really quick and non-stop. But when the freeze occurred, after let it sit for about 5 seconds, the pop out continue to work again. So I believe computer speed could have played a role in the freezing issue. 

Also, I only tested the issue on direct connect touch monitor and not on Spacedesk. The latency and network usage of Spacedesk could also be the cause of your issue. I'll try to reproduce it and see if I can find the root cause and will keep you update.

Stanley
